after/ftplugin/tex.vim

```diff
@@ -0,0 +1,2 @@
+" Function/augroup definition
```

Comment:
Can you update the comment to describe where it jumps to? I'm not very familiar with TeX/LaTeX, but based on vimtex's documentation, something like "Section boundary" should be correct?

It should be a short and the to the point description, and gets used in the generated table in the README (use `tbl` to generate).

Thanks!

Reply:
Right, I missed that part.
More likely "Beginning of a section".
Vimtex tries to define all `][`, `]]`... mappings for "beginning and end of next/previous section", but in the end it only jumps to `\end{documents}` if no "end of section" was found.

Reply:
Cheers, thanks!